internal/buildscripts/packaging/installer/install.ps1

```diff
@@ -64,6 +64,18 @@
     (OPTIONAL) Whether to install and configure fluentd to forward log events to the collector (default: $true)
     .EXAMPLE
     .\install.ps1 -access_token "ACCESSTOKEN" -with_fluentd $false
+.PARAMETER with_dotnet_tracing
+    (OPTIONAL) Whether to install and configure .NET tracing to forward .NET application traces to the local collector (default: $false)
+    .EXAMPLE
+    .\install.ps1 -access_token "ACCESSTOKEN" -with_dotnet_tracing $true
+.PARAMETER signalfx_service_name
```

Comment:
Q.: what are the plans when there is more than on app on the box? The typical case would be multiple apps on Windows IIS.

Reply:
Yes, SFx.NET does that. I'm thinking about the wizard in this case: how we communicate that to the user?

Reply:
If someone figures it out, then I guess the same solution could be applied to this issue >
https://github.com/signalfx/splunk-otel-collector/pull/2068#discussion_r986215896

Reply:
For the wizard, we can provide an option to customers to set the service name as they would like upfront. It would definitely help to allow explicit naming per app instrumented on the host/VM - based on the above, is this not possible today with both the SFx and OTel .NET libraries?

Reply:
Explicitly setting is no problem at all. I guess @pjanotti just meant that it's not basically 0 config (you just setup the collector and instrumentation and you are good to go). While in Windows it's very common to have multiple services on one instance, the default behaviour just doesn't work in UX wise (if the wizard is asking to setup global service name).

Reply:
The goal with 0 config here is to get started with APM with the first deploy of the Collector (as OOTB as possible) with no explicit step or configuration for the instrumentation agents at that point. Customers should be able to customize (exclude processes, set names) if needed thereafter with recommended documentation for steps on the machine. Ideally, we would like such customization to move to the UI in the future but that is not possible today. Circling back to our discussion - makes sense, I see this as an opportunity for us to remove the global service name from the wizard and letting the .NET infer it to avoid confusion. 

@pjanotti @RassK is there any documentation on how can customers explicitly configure names per app vs the env var for the whole machine?

Reply:
We currently instruct users to set process scoped environment variables. In the advanced section we also mention how to do it differently (through a config file).

Refs >
* https://github.com/signalfx/signalfx-dotnet-tracing#instrument-a-net-application-on-windows
* https://github.com/signalfx/signalfx-dotnet-tracing/blob/main/docs/advanced-config.md#advanced-configuration

Reply:
Which brings a rough idea to me. If UI would allow to select root paths of the interested applications, it can plant a config file there (unfortunately for .NET only SignalFx / Splunk variables work, as these can be picked up alternatively). 
The second issue is devops pipelines, there could be a config option that removes all items in the development path, before deploying a new version. That means the most effective UX / UI would be in the tool itself (for eg a Octopus plugin).